includes/html/print-map.inc.php

```diff
@@ -315,10 +334,25 @@
 $nodes = json_encode(array_values($devices_by_id));
 $edges = json_encode($links);
 
+array_multisort(array_column($devices_by_id, 'label'), SORT_ASC, $devices_by_id);
+
 if (count($devices_by_id) > 1 && count($links) > 0) {
 ?>
 
+<form name="printmapform" method="post" action="" class="form-horizontal" role="form">
```

Comment:
Becuse this needs to be a GET?

Reply:
if i change it to GET:
URL will be: `/map?highlight_node=<device_id>`. In classical PHP this is valid. Here it doesn't work.
It has to be changed to `/map/highlight_node=<device_id>` which works well

Reply:
don't know what to do to get it to this here

Reply:
@Jellyfrog some ideas?

Reply:
is solved it with javascript. Maybe later a better way will be implemented :-)